test/yagna/helpers/activity.py

```diff
@@ -0,0 +1,73 @@
+"""Activity helpers."""
+
+import os
+from pathlib import Path
+
+from goth.runner import Runner
+
+
+def vm_exe_script(runner: Runner, output_file: str):
+    """VM exe script builder."""
+
+    output_path = Path(runner.web_root_path) / output_file
+    if output_path.exists():
+        os.remove(output_path)
+
+    web_server_addr = f"http://{runner.host_address}:{runner.web_server_port}"
+
+    return [
+        {"deploy": {}},
+        {"start": {}},
+        {
+            "transfer": {
+                "from": f"{web_server_addr}/scene.blend",
+                "to": "container:/golem/resource/scene.blend",
+            }
+        },
+        {
+            "transfer": {
+                "from": f"{web_server_addr}/params.json",
+                "to": "container:/golem/work/params.json",
+            }
+        },
+        {"run": {"entry_point": "/golem/entrypoints/run-blender.sh", "args": []}},
+        {
+            "transfer": {
+                "from": f"container:/golem/output/{output_file}",
+                "to": f"{web_server_addr}/upload/{output_file}",
+            }
+        },
+    ]
+
+
+def wasi_exe_script(runner: Runner, output_file: str = "upload_file"):
+    """WASI exe script builder."""
```

Comment:
```suggestion
    """Create a WASI exe script for running a WASI tutorial task.

    If `output_file` is not specified a default of `upload_file` will be used as the
    result file name.
    """
```

Reply:
I'd add this, but PR is merged. Will create a new one with this